### DISCUSSION TITLE
Advent Calendar 2021で記事タイトルの表示文字数が少ない（可能性がある）

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#73] [要望・アイデア・不具合]
Advent Calendar 2020までは、カレンダーの下部にリスト形式で記事タイトルが表示されており、カレンダーでは長くて表示しきれないタイトルもそちらで確認することができました。

<img width="1153" alt="スクリーンショット 2021-11-28 21 57 12" src="https://user-images.githubusercontent.com/20086673/143768749-515c8c9a-5a31-46c0-9699-5e62e74b2032.png">

https://qiita.com/advent-calendar/2020/typescript

Advent Calendar 2021ではそれがなくなっているうえに文字サイズも大きくなっているため、カレンダーページ上で確認できるタイトルが短くなっています。

<img width="1153" alt="スクリーンショット 2021-11-28 21 57 19" src="https://user-images.githubusercontent.com/20086673/143768758-9d31da83-9e6a-48b4-a44e-02a236f5aec9.png">

https://qiita.com/advent-calendar/2021/typescript

実際に記事が投稿されたら表示されるようになるという仕様であればコメントが確認できないだけで済みますが、そうではない場合、短い文字数で読むかどうか判断しなければなりません。

## FYI

- スクリーンショットはウインドウの横幅1800pxの状態で撮影

**Comment by kskwtnk** [comment]
@munierujp 

ご意見ありがとうございます！

こちら社内でIssueが立っておりまして、明日か明後日にはリリース予定です！

※ https://github.com/increments/qiita-discussions/discussions/19 こちらのDiscussionでも少し触れているのですが、社内にあるIssueとこちらでの情報共有が完全に同期しておらずすみません！

**Comment by kskwtnk** [comment]
@munierujp 

こちらの件ですが、対応してリリース完了しました！

**Reply by munierujp** [reply]
いいですね 👍 

<img width="1051" alt="スクリーンショット 2021-12-01 12 25 08" src="https://user-images.githubusercontent.com/20086673/144166547-6115c937-7660-4af8-8739-62cd4358f760.png">

**Reply by kskwtnk** [reply]
ありがとうございます！
